src/CommunityPatch/Options/OptionsStore.cs

```diff
@@ -0,0 +1,17 @@
+namespace CommunityPatch {
+
+  public abstract class OptionsStore {
+
+    public abstract void Save();
+
+    public abstract void Set<T>(string key, T value);
+
+    public abstract void Set<T>(string ns, string key, T value);
```

Comment:
Can we name these variables to be clearer for API purposes?

Reply:
`ns` to `namespace`?

Reply:
Yep

Reply:
`namespace` is a language keyword.

Reply:
Discussed in Discord, will add xml docs to flesh out description of parameters.